docker-compose.yml

```diff
@@ -507,6 +507,8 @@ services:
     ports:
       - 8086:8086
 
+  # This container is just to transfer Replicator jars to the Connect worker
+  # It is not used as a Connect worker
   replicator:
```

Comment:
If it's being used other than to extract JARs I would probably change the service/host/container name to something more indicative, i.e. not `replicator`.  

I'd also drop the environment clause entirely. 

For the `command:`, `sleep infinity` is my personal preference instead of `tail -f /dev/null`, it's more expressive/easier to grok at a glance what it's doing.

Reply:
@rmoff excellent suggestions, I have addressed all 3 points